test/documenter/documenter.js

```diff
@@ -27,69 +27,69 @@ const Colibri = require('../../src/main');
 const Documenter = Colibri.Documenter;
 const General = Colibri.General;
 
-let file_vhdl = __dirname + path_lib.sep + "resources" + path_lib.sep + "vhdl" 
-                        + path_lib.sep + "test_0.vhd";
+// let file_vhdl = __dirname + path_lib.sep + "resources" + path_lib.sep + "vhdl" 
+//                         + path_lib.sep + "test_0.vhd";
 
-let code_vhdl = fs.readFileSync(file_vhdl, {encoding:'utf8'});
-let lang_vhdl = General.LANGUAGES.VHDL;
-let comment_symbol_vhdl = "!";
+// let code_vhdl = fs.readFileSync(file_vhdl, {encoding:'utf8'});
+// let lang_vhdl = General.LANGUAGES.VHDL;
+// let comment_symbol_vhdl = "!";
```

Comment:
This file is almost totally commented...

Reply:
Done.

Documenter tests will be fixed when the Documenter output is stable.